src/pages/repository/config/rootTabConfig.ts

```diff
@@ -41,6 +43,12 @@ export const rootTabConfig = [
     Icon: FiGitBranch,
     getPath: (id: string, _?: string) => `/repository/${id}/forks`
   },
+  {
+    title: 'Deployments',
+    Component: DeploymentsTab,
+    Icon: GrDeploy,
```

Comment:
for very weird reason this icon has a static color even when the active tab is not deployments. can you look into it?
<img width="540" alt="Screenshot 2023-12-18 at 8 34 34 PM" src="https://github.com/labscommunity/protocol-land/assets/57343520/57d3eedd-480d-46a2-b819-1dca3d1e67e5">